jobs/gorouter/templates/gorouter.yml.erb

```diff
@@ -80,13 +80,6 @@ def max_request_header_bytes
   if_p("router.max_request_header_kb") do |kb|
```

Comment:
Now that the max_request_header_kb has a default value (nice job changing that, it was the correct thing to do), we no longer need to use `if_p` here. `if_p` is only if there is a chance the property might not be set. Instead you can always grab the property with `kb = p("router.max_request_header_kb")`